tox.ini

```diff
@@ -28,7 +28,14 @@ commands =
     python setup.py -q bdist_wheel sdist
     twine upload --skip-existing dist/*
 
-[testenv:pep8]
+[testenv:lint]
 basepython = python2.7
-deps = flake8
-commands = flake8 twine/ tests/ docs/
+deps =
+    flake8
+    check-manifest
+    readme_renderer
+commands =
+    flake8 twine/ tests/ docs/
+    -check-manifest -v
```

Comment:
I'm new to check-manifest, can you share the reasoning behind having the leading `-` please?

Reply:
Check manifest ensures that all of the relevant files needed for build are included in the sdist (this is super important for downstream packagers). Right now this fails - so the leading `-` tells Tox to ignore the error but still report the output. We'll do a follow-up PR to address the problems, then remove the leading `-`.